src/guides/v2.3/config-guide/log/log-db.md

```diff
@@ -1,6 +1,6 @@
 ---
 group: configuration-guide
-title: Example&mdash;logging database activity
+title: Example - logging database activity
```

Comment:
Revert this. Long term, it would be better if the title were changed so that that there's no em dash.

Reply:
**done**